### DISCUSSION TITLE
Snippets!

### DISCUSSION THREAD

**Discussion by aviv82** [lab-antwerp-1/agile-group-1#69] [Ideas]
# useful bits of code discovered through the creation process

**Comment by aviv82** [comment]
# :root
## description:
 sets general values for the entire .css document:

## code: 

`:root {
--header-font-family: eu;
--paragraph-font: something to decide;
--wine-light-color: #f3e2e3;
--wine-mid1-color: #8a5658;
--wine-mid2-color: #a04048;
--wine-dark-color: #582327;
}`

## comments:
should appear at top of document

**Comment by aviv82** [comment]
# @font-face

## description:
adds new fonts and font families

## code:

` @font-face {
font-family: eu;
src: url(/assets/euphorigenic.ttf);
}`

## comments:
should appear at top of document

**Comment by colevandersWands** [comment]
these are great! Do you mind sending them as PRs to the class repo?

**Reply by aviv82** [reply]
of course!
